prow/plugins/blockade/blockade_test.go

```diff
@@ -296,13 +296,12 @@ func TestHandle(t *testing.T) {
 
 	for _, tc := range tcs {
 		expectAdded := []string{}
-		fakeClient := &fakegithub.FakeClient{
-			RepoLabelsExisting: []string{labels.BlockedPaths, otherLabel},
-			IssueComments:      make(map[int][]github.IssueComment),
-			PullRequestChanges: make(map[int][]github.PullRequestChange),
-			IssueLabelsAdded:   []string{},
-			IssueLabelsRemoved: []string{},
-		}
+		fakeClient := fakegithub.NewFakeClient()
+		fakeClient.RepoLabelsExisting = []string{labels.BlockedPaths, otherLabel}
+		fakeClient.IssueComments = make(map[int][]github.IssueComment)
```

Comment:
Nit: The two maps now get initialized in the constructor and the slices do not need it